app/controllers/api/stateless/braintree/payment_methods_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module Api
+  module Stateless
+    module Braintree
+      class PaymentMethodsController < StatelessController
+        before_filter :authenticate_request!
+
+        def index
+          @payment_methods = PaymentHelper::Braintree.payment_methods_for_member(@current_member)
+        end
+
+        def destroy
+          @payment_method = PaymentHelper::Braintree.payment_method_for_member(member: @current_member, id: params[:id])
+
+          result = begin
+                     ::Braintree::PaymentMethod.delete(@payment_method.token)
+                   rescue ::Braintree::NotFoundError
+                     @payment_method.destroy
+                   end
+
+          @payment_method.destroy if result.success?
+          render json: { success: true }
```

Comment:
I've added a todo. I can't see why it would return false. It's more likely that `destroy` will raise, with the client getting a `422`.